system/Database/Database.php

```diff
@@ -124,8 +124,10 @@ protected function parseDSN(array $params): array
      * Initialize database driver.
      *
      * @param array|object $argument
+     *
+     * @return BaseConnection|BaseUtils|Forge
      */
-    protected function initDriver(string $driver, string $class, $argument): object
+    protected function initDriver(string $driver, string $class, $argument)
```

Comment:
Why remove this? It's consistent with the docblock and natively PHP will process as the much-broader `mixed` otherwise.

Reply:
Yes, we don't need to remove it.
Reverted.